e2e-tests/functions

```diff
@@ -1233,7 +1233,7 @@ function check_pxc_liveness() {
 	wait_cluster_consistency "${cluster}" "${cluster_size}"
 
 	wait_for_running "${cluster}-pxc" "${cluster_size}"
-
+	sleep 60
```

Comment:
Why is this needed if we are waiting for running pods and cluster status? Doesn't it seem to long?